src/components/LeaderBoard/Leaderboard.jsx

```diff
@@ -30,15 +31,32 @@ function useDeepEffect(effectFunc, deps) {
 const LeaderBoard = ({
   getLeaderboardData,
   getOrgData,
+  getMouseoverText,
   leaderBoardData,
   loggedInUser,
   organizationData,
   timeEntries,
   isVisible,
   asUser,
+  totalTimeMouseoverText,
 }) => {
   const userId = asUser ? asUser : loggedInUser.userId;
   const isAdmin = ['Owner', 'Administrator', 'Core Team'].includes(loggedInUser.role);
+  const isOwner = ['Owner'].includes(loggedInUser.role);
+
+  const [mouseoverTextValue, setMouseoverTextValue] = useState(totalTimeMouseoverText);
+
+  useEffect(() => {
+    getMouseoverText();
+  }, [totalTimeMouseoverText]);
+
+  useEffect(() => {
```

Comment:
I would suggest to make just one useEffect because they have the same dependency anyway

Reply:
Thank you! Fixed.